reference/dic_tags.rst

```diff
@@ -540,14 +540,16 @@ kernel.event_listener
 
 **Purpose**: To listen to different events/hooks in Symfony
 
-This tag allows you to hook your own classes into Symfony's process at different
-points.
+During the execution of a Symfony application, different events are triggered.
+This tag allows you to *hook* your own classes into any of those events.
```

Comment:
I like your proposal but I'm stuck trying to implement it. Do you have some suggestion for the new explanation? Thanks!